torchgeo/samplers/utils.py

```diff
@@ -39,6 +44,7 @@ def get_random_bounding_box(
     Args:
         bounds: the larger bounding box to sample from
         size: the size of the bounding box to sample
+        sample_mode: whether to sample in pixel space or CRS unit space
```

Comment:
I think this explanation is a little unclear. `sample_mode` just dictates whether `size` is in pixels or CRS units, right?